episodes/01-rstudio-intro.Rmd

```diff
@@ -66,6 +66,19 @@ Please ensure you have the latest version of R and RStudio installed on your mac
 - [Download and install the latest version of R here](https://www.r-project.org/)
 - [Download and install RStudio here](https://www.rstudio.com/)
 
+:::::::::::::::::::::::::::::::::::::::: instructor
+
+- If your workshop includes the [Introduction to Geospatial Concepts](https://datacarpentry.org/organization-geospatial/) lesson, learners will have
+  just been introduced to RStudio in the context of the overall Geospatial
+  software landscape.
+- Have your learners open RStudio and follow along as you explain each pane. Make sure that your RStudio environment is the default so learners can follow along.
+- Be sure to explain how to execute code from the script window, whether you're
+  using the Run button or the keyboard shortcut.
+- Learners will be using several libraries in the next lesson, so be sure to
+  introduce what a library is and how it is installed.
```

Comment:
```suggestion
- Learners will be using several packages in the next lesson, so be sure to
  introduce what a package is and how it is installed.
```

Apologies for the nit-pick, but this is something that some members of the R community (and R-core) are rather insistent on -- from [Section 1 of "Writing R Extensions"](https://cran.r-project.org/doc/manuals/R-exts.html#Creating-R-packages):

> A package is not a library. The latter is used in two senses in R documentation.
>
> +  A directory into which packages are installed, e.g. /usr/lib/R/library: in that sense it is sometimes referred to as a library directory or library tree (since the library is a directory which contains packages as directories, which themselves contain directories).
> +  That used by the operating system, as a shared, dynamic or static library or (especially on Windows) a DLL, where the second L stands for ‘library’. Installed packages may contain compiled code in what is known on Unix-alikes as a shared object and on Windows as a DLL. The concept of a shared library (dynamic library on macOS) as a collection of compiled code to which a package might link is also used, especially for R itself on some platforms. On most platforms these concepts are interchangeable (shared objects and DLLs can both be loaded into the R process and be linked against), but macOS distinguishes between shared objects (extension .so) and dynamic libraries (extension .dylib).

Any chance you'd be willing to change "library" to "package" here?